File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Map<Integer, Integer> hm = new HashMap<>();
        for(int num : nums)
        {
            if(!hm.containsKey(num))
            {
                hm.put(num,num);
            }
            else{
                return true;
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        numSet = set(nums)
        longest = 0

        for num in numSet:
            length = 1
            if num-1 not in numSet:
                while num + length in numSet:
                    length +=1
            if longest < length:
                longest = length
        return longest

